metadata-ingestion/src/datahub/ingestion/source/sql/mssql/source.py

```diff
@@ -190,43 +230,31 @@ def handle_sql_variant_as_string(value):
     def _populate_table_descriptions(self, conn: Connection, db_name: str) -> None:
         # see https://stackoverflow.com/questions/5953330/how-do-i-map-the-id-in-sys-extended-properties-to-an-object-name
         # also see https://www.mssqltips.com/sqlservertip/5384/working-with-sql-server-extended-properties/
-        table_metadata = conn.execute(
-            """
-            SELECT
-              SCHEMA_NAME(T.SCHEMA_ID) AS schema_name,
-              T.NAME AS table_name,
-              EP.VALUE AS table_description
-            FROM sys.tables AS T
-            INNER JOIN sys.extended_properties AS EP
-              ON EP.MAJOR_ID = T.[OBJECT_ID]
-              AND EP.MINOR_ID = 0
-              AND EP.NAME = 'MS_Description'
-              AND EP.CLASS = 1
-            """
+        path_to_table_description_query = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)),
+            "sql_queries/table_description_query.sql",
```

Comment:
we should have a constant at the top of the file like `SQL_QUERIES_DIR`

this should use the `pathlib` library

this code can then become `query = (SQL_QUERIES_DIR / "filename.sql").read_text()`